JSON.md

```diff
@@ -1,14 +1,27 @@
 # CFEngine Build JSON file format
 
-The CFEngine Build website and tooling relies on JSON files for different purposes;
+The `cfbs` command line tool uses `cfbs.json` configuration files.
+We call the folder / git repo which has a `cfbs.json` file (and optionally other files and folders) a _project_, and the end result after combining modules in a project into policy a _policy set_.
+The _policy set_ is what you deploy to your CFEngine hub / policy server, in `/var/cfengine/masterfiles`.
 
-* The project file (`cfbs.json`) generated by running `cfbs init` when you start a new project
-* The index of all available modules which is available on GitHub
-* A `cfbs.json` file with necessary metadata for adding a module using URL (not from index)
+See the [README](https://github.com/cfengine/cfbs/blob/master/README.md) for general documentation on the `cfbs` tool and its commands.
+This file is specifically about the `cfbs.json` file format, it should serve as detailed and useful information for more advanced users who are making their own modules or contributing to CFEngine Build.
 
-All these 3 share 1 standard format, commonly called `cfbs.json`.
+There are 3 types of `cfbs.json` files:
 
-(There is also [`versions.json`](https://github.com/cfengine/cfbs-index/blob/master/versions.json), but this specification is not about that file).
+* For building a policy set you create a project with type `policy-set`.
+  This is the default when running `cfbs init`, and what most users encounter when first using the tool and CFEngine.
+  You then need to use the `build` key to specify which modules to use in `cfbs build`.
```

Comment:
```suggestion
  You then need to use the top-level `build` key to specify which modules to use in `cfbs build`.
```

Reply:
Hmm, to me it seems to be a bit more tedious / harder to read if that is repeated every time. Probably I'll try to accept some, but not all, of the suggestions.